packages/orchestrator-ui-components/src/rtk/api.ts

```diff
@@ -13,6 +13,11 @@ export enum BaseQueryTypes {
     custom = 'custom',
 }
 
+export enum CacheTags {
+    engineStatus = 'engineStatus',
+    testStatus = 'testStatus', // This is a placeholder for now. Having only one messes up the type of the tag in the onCacheEntryAdded function.
```

Comment:
I think our linting rules do not apply to comments. Suggesting to place comments above what needs to be commented. Can can either be above `testStatus` or the entire enum I would say